src/main/java/dk/kb/ds/cumulus/export/ImageUrl.java

```diff
@@ -0,0 +1,29 @@
+package dk.kb.ds.cumulus.export;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ImageUrl {
+
+    public static String makeUrl (String image_url) {
+        String path1[] = image_url.split(".tif");
+        String path2[] = path1[0].split(":/");
+        String path = path2[1].replace("Depot/DAMX/Online_Master_Arkiv", "DAMJP2/online_master_arkiv");
+        image_url = "https://kb-images.kb.dk/" + path + "/full/!345,2555/0/native.jpg";
```

Comment:
This should be a setting, but let's keep it for now to move things forward and make a JIRA instead.